plugins/inputs/win_services/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Plugin: win_services
+Input plugin to report Windows services info: service name, display name, state, startup mode
```

Comment:
Ok, but the sentence will be less descriptive. What about:  Input plugin to report Windows services basic info, including service name, display name, etc.